en/04_Changelogs/5.0.0.md

```diff
@@ -60,6 +61,31 @@ This is a major release and as a result there are a number of breaking API chang
     - `SilverStripe\ORM\Connect\Query`
   - Removed `SilverStripe\ORM\DataList::getGenerator()` (use `getIterator()` instead)
   - Removed the `SilverStripe\ORM\Map_Iterator` class. `SilverStripe\ORM\Map` now uses a generator instead.
+- Removed various API related to SwiftMailer
+  - Removed the following classes and interfaces
+    - `SilverStripe\Control\Email\Mailer`
+    - `SwiftMailer`
+    - `SwiftPlugin`
+  - Removed the following methods from `SilverStripe\Control\Email\Email`, in most instances these have been replaced with equivalent functionality:
+    - `addAttachment()`
+    - `addAttachmentFromData()`
+    - `debug()`
+    - `hasPlainPart()`
+    - `invalidateBody()`
+    - `findPlainPart()`
+    - `generatePlainPartFromBody()`
+    - `getBCC()`
+    - `getCC()`
+    - `getFailedRecipients()`
+    - `getFrom()`
+    - `getReturnPath()`
+    - `getSender()`
+    - `getSwiftMessage()`
+    - `getTo()`
+    - `setSwiftMessage()`
+    - `setFailedRecipients()`
+    - `BaseURL()`
+    - `IsEmail()`
```

Comment:
> getTo() replaced with to()
This isn't accurate - to() means setTo()

Silverstripe Email::getTo() used to return an array of strings (I think, possibly a string, or maybe something else),   Silverstripe getTo() has been removed.  Email::getTo() was just a wrapper for SwiftMailer SimpleMimeEntity getTo()

However there is still a getTo() method on Email - Symfony Email::getTo() which returns an array of Symfony Address object

So ... I feel this is really too intricate a thing to put in a changelog.  I think saying "in most instances these have been replaced with equivalent functionality" is good enough here.  Realistically very few projects will be using calling methods like `getTo()` because generally they only care about `setTo()` because they just want to send an email.  Possibly it'll cause some unit tests in projects to fail, but that will be easy to fix.

Looking at this list, only addAttachment() and addAttachmentFromData() will be relevant - I'm thinking that it's actually better here to just alias them to the new method sames, similar to what I did with the `setTo()` -> `to()` alias - this means less likely to have silent regressions on upgrade

Reply:
Apologies, I got `get` and `set` mixed up - the `getTo()` method, along with a number of others here, _are not being removed from the class's public API_.  Sure we're not implementing these methods _directly_ in `Email` anymore, but from the developers POV the method is still there. I can still call `$email->getTo()`. Saying we've removed those methods is just going to confuse people, since the methods are still present and callable, and they don't actually have to change anything about how they use them.

For any methods we are keeping (either directly or via inheritance), please remove them from this list.
For any methods are are removing, but which have a replacement (if any) please add "replaced with `theMethod()`" so that developers know what method they should be calling instead.

If any of the methods we're keeping have changed in a substantial way (changed parameters or returned values) please note this in the email section below.

Reply:
Updated